q2_sample_classifier/tests/test_classifier.py

```diff
@@ -537,6 +538,15 @@ def _load_cmc(md_fp, column):
             self.get_data_path('importance.tsv'), sep='\t')
         self.exp_pred = pd.Series.from_csv(
             self.get_data_path('predictions.tsv'), sep='\t', header=0)
+        index = pd.Index(['A', 'B', 'C', 'D'], name='id')
+        self.table_percnorm = qiime2.Artifact.import_data(
+            FeatureTable[PercentileNormalized], pd.DataFrame(
+                [[20.0, 20.0, 50.0, 10.0], [10.0, 10.0, 70.0, 10.0],
+                 [90.0, 8.0, 1.0, 1.0], [30.0, 15.0, 20.0, 35.0]],
+                index=['A', 'B', 'C', 'D'],
```

Comment:
How about

```suggestion
                index=index,
```

since you already defined it once, above.